rpc/handlers.go

```diff
@@ -775,9 +776,70 @@ func getAddInvokeTxCode(err error) int {
 		return jsonrpc.InvalidParams
 	case strings.Contains(err.Error(), "not supported. Supported versions"):
 		return jsonrpc.InvalidParams
-	case strings.Contains(err.Error(), "max_fee must be bigger than 0.\n0 >= "):
+	case strings.Contains(err.Error(), "max_fee must be bigger than 0"):
 		return jsonrpc.InvalidParams
 	default:
 		return jsonrpc.InternalError
 	}
 }
+
+// AddDeclareTransaction relays a declare transaction to the gateway.
+//
+// It follows the specification defined here:
+// https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_write_api.json#L39
```

Comment:
We can use [godoc formatting](https://go.dev/doc/comment#links) here.

Reply:
The godoc format causes linting issues, and the current version is also consistent with the rest of the documentation comments in this file. So I think it makes sense to leave it as is for now.